package.json

```diff
@@ -51,10 +51,15 @@
     "coffee-errors": "^0.8.6",
     "coffee-script": "~1.10.0",
     "csonschema": "^0.5.1",
+    "fs": "0.0.2",
     "glob": "^7.0.3",
+    "jsonlint": "^1.6.2",
     "mocha": "~2.5.1",
+    "node-rest-client": "^1.5.1",
+    "optimist": "^0.6.1",
     "mustache": "~2.2.1",
     "proxyquire": "^1.3.1",
+    "q": "^1.4.1",
```

Comment:
@plroebuck  I believe `node-rest-client` is not being used in our code. There is another library `request` doing the same job.